### ISSUE TITLE
[] operator not supported for strings (with fix)

### ISSUE THREAD

**Issue by rorso** [JamesHeinrich/getID3#264]
If a 'warning' condition is met during analysis, a proper message should be added to the $info['warning'] **array**.

However, function RecursiveFrameScanning() calls decodeMPEGaudioHeader() with an array where this entry is preset as **string**.

```php
	public function RecursiveFrameScanning(&$offset, &$nextframetestoffset, $ScanAsCBR) {
		$info = &$this->getid3->info;
->		$firstframetestarray = array('error'=>'', 'warning'=>'', 'avdataend'=>$info['avdataend'], 'avdataoffset'=>$info['avdataoffset']);
		$this->decodeMPEGaudioHeader($offset, $firstframetestarray, false);
...
->			$nextframetestarray = array('error'=>'', 'warning'=>'', 'avdataend'=>$info['avdataend'], 'avdataoffset'=>$info['avdataoffset']);
			if ($this->decodeMPEGaudioHeader($nextframetestoffset, $nextframetestarray, false)) {
...
```

This triggers the Error '**[] operator not supported for strings**' in PHP 7.1 onwards as soon as there is a 'warning' or 'error' condition to be added.

replace both assignments in `module.audio.mp3.php` near line 1076 with

```php
array('error'=>array(), 'warning'=>array(), 'avdataend'=>$info['avdataend'], 'avdataoffset'=>$info['avdataoffset']);
```

and PHP7 is happy again.

**Comment by StudioMaX** [comment]
Which version are you using? This warning was fixed since v1.9.14 by this commit - https://github.com/JamesHeinrich/getID3/commit/d913ebce1607453d8a7128ea4da1f080af0479ed on 30 Jan 2017.

**Comment by rorso** [comment]
I'm not using this directly. I found this bug in THIS project that references the getID3 library.
[https://github.com/sverhagen/mp3-browser](https://github.com/sverhagen/mp3-browser)

I maybe wrongly assumed the referenced package is synced between the projects. Then you may safely deny this "fix" and I will kick THEM to update their references.

Sorry for causing confusion.
